python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py

```diff
@@ -142,9 +142,10 @@ def __init__(self, index=1, remotes=False):
 
         # Matrix to map hexapod offset to alt/az offset in the focal plane
         # units are arcsec/mm. X-axis is Elevation
+        # Measured with data from AT run SUMMIT-5027, still unverified.
         self.hexapod_offset_scale = [
-            [60.0, 0.0, 0.0],
-            [0.0, 60.0, 0.0],
+            [56.8, 0.0, 0.0],
```

Comment:
Can you update these with the values that Craig provided?

 52.46

and 

50.47

respectively